pkg/logger/s3_log_writer.go

```diff
@@ -18,60 +16,31 @@ type S3LogWriter struct {
 	log0  Logger
 }
 
-const logBucketName = "gitops-run-logs"
+const SessionLogBucketName = "gitops-run-logs"
+const PodLogBucketName = "pod-logs"
 
 func (l *S3LogWriter) L() logr.Logger {
 	return l.log0.L()
 }
 
-func NewInsecureS3LogWriter(id, endpoint string, accessKey, secretKey string, log0 Logger) (Logger, error) {
-	minioClient, err := minio.New(
-		endpoint,
-		&minio.Options{
-			Creds:        credentials.NewStaticV4(accessKey, secretKey, ""),
-			Secure:       false,
-			BucketLookup: minio.BucketLookupPath,
-		},
-	)
-
-	if err != nil {
-		return nil, err
-	}
-
-	if err := minioClient.MakeBucket(context.Background(), logBucketName, minio.MakeBucketOptions{}); err != nil {
-		return nil, err
-	}
-
+func NewS3LogWriter(minioClient *minio.Client, id string, log0 Logger) (Logger, error) {
```

Comment:
✔️ 
```suggestion
func NewS3LogWriter(minioClient *minio.Client, id string, log Logger) (Logger, error) {
```

Reply:
It's @chanwit 's naming preference, there are multiple options for this name, so, you could discuss it later.